pb-crm-ui/src/app/invoicing/components/account-summary-page/account-summary-page.component.ts

```diff
@@ -1,36 +1,26 @@
-import { Component, OnInit } from '@angular/core';
-import {ActivatedRoute, Router} from "@angular/router";
-import {InvoicingRoutes} from "../../invoicing-routing.module";
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {AccountModel} from '@hiddentemple/api-interfaces';
+import {AccountCacheService} from '../../services/account-cache.service';
 
 @Component({
   selector: 'app-account-summary-page',
```

Comment:
When you go to change the name, this is what should change (ie something like `app-account-detail-page`